R/funs.R

```diff
@@ -5,7 +5,7 @@
 #' appropriate SQL for remote tables.
 #'
 #' @param x A numeric vector of values
-#' @param left,right Boundary values (must be scalars).
+#' @param left, right Boundary values (must be scalars).
```

Comment:
This changes the generated file to : 

```
\item{left, }{right Boundary values (must be scalars).}
```

which is not the intent. 

```suggestion
#' @param left,right Boundary values (must be scalars).
```